### DISCUSSION TITLE
KeyError: __schema

### DISCUSSION THREAD

**Discussion by StefanLecho** [schemathesis/schemathesis#2042] [Q&A] [answered]
I've created a new project listchainlinks-with-default-200-ok-response based on this yaml: https://gist.github.com/DavidBiesack/1d0f20f40c5cc3ec7682b00917e7085d.

When running underneath cmd, I'm getting a **KeyError: __schema**. 
I'm having the same result when using this yaml: https://github.com/opticdev/examples/blob/main/apps/bookstore-api/openapi.yml

Could someone please help me to solve it ?

docker run schemathesis/schemathesis run listchainlinks-with-default-200-ok-response --checks all --report --schemathesis-io-token MY_TOKEN --show-trace
Test Execution Error

An internal error occurred during the test run

    Traceback (most recent call last):
      File "/usr/local/lib/python3.12/site-packages/schemathesis/cli/__init__.py", line 1103, in into_event_stream
        yield from runner.from_schema(
      File "/usr/local/lib/python3.12/site-packages/schemathesis/runner/impl/core.py", line 108, in _generate_events
        initialized = events.Initialized.from_schema(
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/usr/local/lib/python3.12/site-packages/schemathesis/runner/events.py", line 69, in from_schema
        operations_count=schema.operations_count if count_operations else None,
                         ^^^^^^^^^^^^^^^^^^^^^^^
      File "/usr/local/lib/python3.12/site-packages/schemathesis/specs/graphql/schemas.py", line 188, in operations_count
        raw_schema = self.raw_schema["__schema"]
                     ~~~~~~~~~~~~~~~^^^^^^^^^^^^
    KeyError: '__schema'

**Comment by Stranger6667** [comment]
Hi, it is a bug, I am going to fix it

**Reply by Stranger6667** [reply]
@StefanLecho what Schemathesis version are you using?

> I'm having the same result when using this yaml: https://github.com/opticdev/examples/blob/main/apps/bookstore-api/openapi.yml

I can't reproduce it, with the command you provided, it should report an error about Open API 3.1.3 not being supported (and it suggests to use an extra CLI option to continue)

**Reply by StefanLecho** [reply]
The output of "docker run schemathesis/schemathesis --version" is "schemathesis, version 3.25.0"

I have the same issue with both YAMLs when using Python instead od Docker.
The output of "python --version" is "Python 3.11.0"

**Reply by Stranger6667** [reply]
Aha, thanks! I'll investigate it

**Reply by Stranger6667** [reply, marked as answer]
The problem will be fixed in the next release, meanwhile you could add `--experimental=openapi-3.1` and it should work fine :) Sorry for the inconvenience

**Reply by StefanLecho** [reply]
I confirm that it works with `--experimental=openapi-3.1`

**Reply by Stranger6667** [reply]
Great! Feel free to reach out if you have any further questions
